server/config/routes.js

```diff
@@ -35,6 +35,9 @@ module.exports = function (app) {
     .get(plurbController.getPlurb)
     .post(plurbController.deletePlurb);
 
+  app.route('/api/plurbs/:googId')
```

Comment:
I feel like this route may be difficult to create on the client side... I'm not 100% sure about the accessibility of the googId on the client side. Perhaps we could make a unique route for this and have the server-side grab the googId?